docusaurus/docs-internals/editor-controls/data-controls/codeMirror.md

```diff
@@ -252,3 +334,126 @@ In the examples below, we specify the language in which we will write the code.
   }
 }
 ```
+
+#### Default value examples
+
+In this example, the `codeMirror` control, with the default value `".brz-button { background-color: purple; }"`, returned the value from the textarea.
```

Comment:
about: "returned the value from the textarea."

you don't return anything, you just set default value for your textarea